tools/docker-compose/ansible/smoke-test.yml

```diff
@@ -0,0 +1,78 @@
+---
+#
+# This is used by a CI check in GitHub Actions and isnt really
+# meant to be run locally.
+#
+# The development environment does some unfortunate things to
+# make rootless podman work inside of a docker container.
+# The goal here is to essentially tests that the awx user is
+# able to run `podman run`.
+#
+- name: Test that the development environment is able to launch a job
+  hosts: localhost
+  tasks:
+    - name: Boot the development environment
+      command: |
+        make docker-compose
+      environment:
+        COMPOSE_UP_OPTS: -d
+      args:
+        chdir: "{{ repo_dir }}"
+
+    # Takes a while for migrations to finish
+    - name: Wait for the dev environment to be ready
+      uri:
+        url: "http://localhost:8013/api/v2/ping/"
+      register: _result
+      until: _result.status == 200
+      retries: 120
+      delay: 5
+
+    - name: Reset admin password
+      shell: |
+        docker exec -i tools_awx_1 bash <<EOSH
+          awx-manage createsuperuser --noinput --username=admin --email=admin@localhost
+          awx-manage update_password --username=admin --password=password
+          awx-manage create_preload_data
```

Comment:
This doesn't seem to be working 100% from the output of the check

```
"stderr": "CommandError: Error: That username is already taken."
```

I'm not sure _which_ of the lines this is coming from.

Reply:
but since the rest of it works, I think that's merely telling us that something here isn't necessary. I think the `update_password` is absolutely necessary, this wouldn't work without that.